tests/e2e/test_e2e_ssp.py

```diff
@@ -111,33 +100,40 @@ def test_ssp_editing_e2e(
     tmp_repo_path = pathlib.Path(tmp_repo_str)
 
     args = setup_for_ssp(tmp_repo_path, test_prof, [test_comp_name], test_ssp_md)
+    remote_url = "http://localhost:8080/test.git"
```

Comment:
Is it worth considering pushing this sorta thing into conftest ?

Reply:
Good idea. I use this in a number of places.